moped-editor/src/views/projects/projectView/ToolbarPlugin.js

```diff
@@ -144,7 +144,12 @@ const ToolbarPlugin = ({ noteAddSuccess, classes }) => {
   const getSelectedButtonProps = (isSelected) =>
     isSelected
```

Comment:
maybe beyond the scope of this issue, but this ternary look like it could live directly inside the sx prop in the associated Button component with a few small changes.